jwql/website/apps/jwql/static/js/jwql.js

```diff
@@ -250,9 +278,9 @@ function update_thumbnails_page(inst, proposal, base_url) {
         }});
 };
 
-/* JS function to construct the URL corresponding to a specific GitHub release
-* @param {String} version_string - The x.y.z version number
-*/
+/* Construct the URL corresponding to a specific GitHub release
+ * @param {String} version_string - The x.y.z version number
+ */
```

Comment:
Also looks like there are some extra spaces at the start of lines 282 and 283

Reply:
@laurenmarietta I think these are supposed to be there?

Reply:
@laurenmarietta Actually this did make me realize there was a small discrepancy between this docstring and the others.